test/config.go

```diff
@@ -11,19 +11,20 @@ import (
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v3"
 
+	"github.com/livekit/egress/pkg/config"
 	"github.com/livekit/egress/pkg/service"
 	"github.com/livekit/protocol/egress"
 	"github.com/livekit/protocol/utils"
 	lksdk "github.com/livekit/server-sdk-go"
-
-	"github.com/livekit/egress/pkg/config"
 )
 
-type Config struct {
```

Comment:
`Context` usually has a fairly specific meaning in Go, so this was a bit confusing to me initially. Would TestPlan or TestSetup make sense here?